shared/src/main/scala/coop/rchain/diag/Tracing.scala

```diff
@@ -0,0 +1,8 @@
+package coop.rchain.diag
+
+import coop.rchain.metrics.Metrics
+
+object Tracing {
+  val CasperMetricsSource: Metrics.Source = Metrics.Source(Metrics.BaseSource, "casper")
+  val RuntimeManagerMetricsSource         = Metrics.Source(CasperMetricsSource, "runtime-manager")
```

Comment:
Both of these does not belong to shared project. It should be moved to the project where are used.

Reply:
I moved it here because to my understanding - we might need base tracing source from `parent` project, and we don't have access to source defined in that project.

Reply:
I removed that package completely